### DISCUSSION TITLE
Start Menu relocated from middle to left side of window

### DISCUSSION THREAD

**Discussion by Seneschal974** [Open-Shell/Open-Shell-Menu#1665] [Q&A] [answered]
In version .490 the Open-Shell Menu is in the middle of the window instead of the left side of the window as in previous versions. Is there anyway to relocate it to the left side of the window? Thanks.

**Comment by ge0rdi** [comment, marked as answer]
`Open-Shell` is replacing original start button/menu, thus start menu is always located where start button is.

In Windows 11 you can switch between _left_ and _center_ aligned taskbar.
Right-click on taskbar, select `Taskbar settings` and then `Taskbar behavior` -> `Taskbar alignment`.

**Reply by Seneschal974** [reply]
Thanks. That works. ( I had another setting preventing the menu from going all the way left, as well. Corrected that and your suggestion worked.) I would still like to be able to have taskbar in the center and start menu on the left--perhaps an option in the future. How do I give you credit for the answer? When I click on "answer" in my comment, it gives me credit for the answer, not you. Thanks again.

**Reply by ge0rdi** [reply]
You can click `Mark as answer` button on reply that answered your question.

**Reply by Seneschal974** [reply]
Got it. Done. Thanks.
